sdcm/utils/replication_strategy_utils.py

```diff
@@ -0,0 +1,42 @@
+import re
+
+from contextlib import ContextDecorator
+from dataclasses import dataclass
+from typing import List, Callable
+
+from sdcm.cluster import BaseNode
+
+
+@dataclass
+class KeyspaceReplicationStrategy:
+    keyspace: str
+    strategy: str
+
+
+class temporary_replication_strategy_setter(ContextDecorator):  # pylint: disable=invalid-name
```

Comment:
because it is not a class - it's ContextManager. Style was taken from python documentation. See: https://docs.python.org/3/library/contextlib.html#using-a-context-manager-as-a-function-decorator

Reply:
i understand that, and need to confirm lint will not fail with this format

Reply:
it passes pre-commit hooks - see `disable=invalid-name` pylint setting